src/controllers/TodoController.js

```diff
@@ -20,9 +20,11 @@ class TodoController {
   }
 
   static async update(req, res) {
+    const { value } = req.body;
```

Comment:
Lets provide req.body as 'post' to service, where we can have one or two our fields